src/components/SVGIcon.js

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 
-const getViewBox = name => {
+const getViewBox = (name) => {

```

Comment:
is it just a formatting change? if yes revert it

Reply:
I'd like to keep formatting consistent